src/main/java/io/civex/LoginSystem/Listeners/Join.java

```diff
@@ -34,5 +36,9 @@ public void onJoin(PlayerJoinEvent event)
         }
 
         plugin.checkIfUsersShouldBeOnClock(0);
+
+        if (plugin.getConfig().getBoolean("discord.events.queue-empty", false) && beforeSize != 0 && plugin.getHighestQueuePos() == 0) {
```

Comment:
If someone with a perm to bypass the queue joins the game when the server is full but the queue is empty, The `queue is empty` message pops up in Discord. Might be a good idea to check the perm here so we don't get a spam of queue empty messages.